tools/Make/board/qemu_virt_aarch64.mk

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright 2023, UNSW
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+PLATFORM := arm
+
+BLK_DRIV_DIR := virtio
+GPU_DRIV_DIR := virtio
+NET_DRIV_DIR := virtio
+ETH_DRIV := eth_driver_virtio.elf
+TIMER_DRIV_DIR := ${PLATFORM}
+UART_DRIV_DIR := ${PLATFORM}
+
+CPU := cortex-a53
+QEMU := qemu-system-aarch64
+QEMU_ARCH_ARGS := -machine virt,virtualization=on -cpu cortex-a53 \
```

Comment:
Isn't this missing the amount of RAM? `-m 2GB`?

Reply:
It's determined by the board, seL4 determines the amount of RAM at build-time which means you can't really use the parameter unfortunately.